client/my-sites/email/email-management/home/utils.js

```diff
@@ -123,6 +125,22 @@ export function resolveEmailPlanStatus( domain, emailAccount, isLoadingEmails )
 			return errorStatus;
 		}
 
+		// When users have registered a domain with us, we let them purchase Google Workspace
+		// before the domain provisioning has finished. However, the user won't see any mailboxes
+		// in the email management until the domain provisioning has finished. To avoid confusion,
+		// we display a warning under these conditions.
+		if (
+			isRegisteredDomain( domain ) &&
+			! domain.hasWpcomNameservers &&
+			getGSuiteSubscriptionStatus( domain ) === 'unknown'
+		) {
+			return {
+				statusClass: 'warning',
+				icon: 'info',
+				text: translate( 'Configuring domain…' ),
```

Comment:
Would it be clearer to the user if we were `Configuring your mailboxes` , or `Configuring Google Workspace`?